internal/sftp/goclient_test.go

```diff
@@ -227,8 +227,7 @@ func TestGoClient(t *testing.T) {
 			},
 			wantErr: fmt.Sprintf(
 				"SSH: connect: dial tcp %s:%s: connect: connection refused",
-				badHost, badPort,
-			),
+				badHost, badPort),
```

Comment:
Why is this change necessary?

Reply:
its a result of lint i think

Reply:
I just ran the linter on main and it didn't change this. :confused: